src/Agent.Worker/Release/Artifacts/Definition/TfvcArtifactDetails.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Microsoft.VisualStudio.Services.Agent.Worker.Release.Artifacts.Definition
+{
+    public class TfVCArtifactDetails : IArtifactDetails
```

Comment:
I believe the general .net naming convention is: capitalize both for two letter acronym, capitalize first letter only for three+ letter acronym.

It would be nice to have naming consistency here. One of the struggles I had naming the TFVC source related classes, is that separate command implementations exist depending on platform. I went with "TfsVC" for the platform agnostic source provider class, "***TF***CommandManager" for the "***tf***.exe" wrapper, and "TeeCommandManager" for the TEE CLC wrapper.

Notice, the "TfsGitSourceProvider" follows the same prefix.

Furthermore, these names are consistent with the WellKnownRepositoryTypes enum values in the build web API dll: `TfsGit` and `TfsVersionControl`